modules/dna_rna_tools.py

```diff
@@ -0,0 +1,92 @@
+def check_input(seqs: list, operation: str):
+    """
+    Check input for some of the most common errors
+    arguments:
+    - seqs (list): a list of sequences
+    - operation (str): operation name
+    return
+    - no return
+    """
+    from modules.dna_rna_constants import dna_rna_alphabet
+    for seq in seqs:
+        if type(seq) != str:
+            raise ValueError('Invalid input: all sequences must be of type str!')
+        t_present = False
+        u_present = False
+        for i in seq:
```

Comment:
Тут все таки лучше не `i`, это же не счетчик какой-то или индекс, это нуклеотид. 
```suggestion
        for nucl in seq:
```